java/java.editor.base/src/org/netbeans/modules/java/editor/base/semantic/SemanticHighlighterBase.java

```diff
@@ -944,60 +877,35 @@ public Void visitMethodInvocation(MethodInvocationTree tree, EnumSet<UseTypes> d
                 }
             }
             
-            if (!handled) {
```

Comment:
Unlike the previous impl, the method's identifier will be processed by `handlePossibleIdentifier` during `scan()` at line 887. Is it OK ?